README.md

```diff
@@ -1,57 +1,215 @@
-# Project Name
+# Extracting, transforming, and loading data with ACI
 
-(short, 1-3 sentenced, description of the project)
+In this sample, we're trying to do some basic analysis on the [HappyDB](https://github.com/rit-public/HappyDB) dataset, by doing word count and drawing word cloud, to highlight the words appear more frequently in the people's happy moments.
 
-## Features
+**Table of content**
 
-This project framework provides the following features:
+[Foreword](#foreword)
 
-* Feature 1
-* Feature 2
-* ...
+[Build Docker Images](#build-docker-images)
 
-## Getting Started
+[Create Azure Storage Account and File Share](#create-azure-storage-account-and-file-share)
 
-### Prerequisites
+[Deploy Azure Components](#deploy-azure-components)
 
-(ideally very short, if any)
+[Check the Demo](#check-the-demo)
 
-- OS
-- Library version
-- ...
+* [View Demo in Web](#view-demo-in-web)
 
-### Installation
+* [Create Index and Query Data in Azure Shell](#create-index-and-query-data-in-azure-shell)
 
-(ideally very short)
+[References](#references)
 
-- npm install [package name]
-- mvn install
-- ...
+## Foreword
 
-### Quickstart
-(Add steps to get up and running quickly)
+Making ETLing simpler using containers as a plug and play model.
```

Comment:
Make ETLing simpler by using Azure Container Instances as a plug and play model.

Reply:
updated